recipe/meta.yaml

```diff
@@ -1,27 +1,25 @@
-
 {% set name = "xbpch" %}
-{% set version = "0.3.2" %}
-{% set sha256 = "58e943de3423b0d105c0c392315ad1596f91c9601535ac5d118ffb1147387efe" %}
+{% set version = "0.3.3" %}
+{% set sha256 = "cfb4df928792614cf067256161543dfbfb19013b8f140eb4ba3a90fd5fed575e" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   build:
-    - python
-    - setuptools
+    - python >=3
```

Comment:
I'm assuming that the py2k failures before was b/c of some Python 3 only syntax used in the module.
@darothen can you confirm that?